### PR TITLE
Adding an Image management workflow

### PR DESCRIPTION
**Pull request by etsauer**
This resolves https://trello.com/c/lH5KvYsD/50-apponb-5-design-and-document-an-image-supply-chain-workflow

@sabre1041 @JaredBurck @mwitzenm

### CODE REVIEW

**Comment by etsauer** [comment]
[APPONB-5: Design and Document an Image Supply Chain Workflow](https://trello.com/c/lH5KvYsD/50-apponb-5-design-and-document-an-image-supply-chain-workflow)

File: image-management/applier/templates/docker-build-git.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: ${BASE_IMAGE_STREAM}
+          namespace: openshift
+      type: Source
+    successfulBuildsHistoryLimit: 5
+    triggers:
+    - github:
+        secret: LaKb2XCE1Uz0GuVc100F
```
Comment (sabre1041) [inline image-management/applier/templates/docker-build-git.yml]: Generate value in template?

File: image-management/applier/templates/docker-build-git.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      type: GitHub
+    - generic:
+        secret: D-WZT8ZtZMg7V3g1y3hY
```
Comment (sabre1041) [inline image-management/applier/templates/docker-build-git.yml]: Generate value in template?

File: image-management/applier/templates/rolebinding.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - kind: ServiceAccount
+    name: builder
+    namespace: image-management
+  userNames:
+  - system:serviceaccount:${BUILDER_NAMESPACE}:builder
+parameters:
+- name: IMAGE_NAMESPACE
+  value: openshift
+- name: BUILDER_NAMESPACE
+  value: image-builds
```
Comment (sabre1041) [inline image-management/applier/templates/rolebinding.yml]: Value should be `image-management`

**Comment by trevorquinn** [comment]
BEFORE: "The purpose of this project is to be used as a bootstrap for an custom **based** image build workflow."

AFTER: "The purpose of this project is to be used as a bootstrap for an custom base image build workflow."

**Comment by trevorquinn** [comment]
BEFORE: "The purpose of this project is to be used as a bootstrap for an custom **based** image build workflow."

AFTER: "The purpose of this project is to be used as a bootstrap for an custom **base** image build workflow."

**Comment by etsauer** [comment]
@sabre1041 @trevorquinn pushed updates.

**Pull request merged**